### PR TITLE
Hide collaborative tag input when empty

### PR DESCRIPTION
**Pull request by nickvergessen**
Checkbox 1.1 from #1465 

@jancborchardt @MorrisJobke 
### No tags

> files_-_nextcloud_-_2016-09-20_18 08 44
### Has tags

> files_-_nextcloud_-_2016-09-20_18 09 20

### CODE REVIEW

File: core/img/actions/tag.svg [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg xmlns="http://www.w3.org/2000/svg" height="16" width="16" version="1.0">
```
Comment (MorrisJobke) [inline core/img/actions/tag.svg]: Please add `viewBox="0 0 16 16"` for proper scaling in Firefox ;)

**Review by MorrisJobke — CHANGES_REQUESTED** [review]

**Review by MorrisJobke — APPROVED** [review]

File: core/img/actions/tag.svg (hunk 1 above)
Comment (MorrisJobke) [inline core/img/actions/tag.svg]: Clicked the wrong button

Comment (nickvergessen) [inline core/img/actions/tag.svg]: Fixed

**Comment by MorrisJobke** [comment]
Tested and works (even if IE and Firefox) ;)

**Comment by jancborchardt** [comment]
Two things we need to fix:
- [ ] »Tags« action should have some padding and space to the left to differentiate from file info
- [ ] when the input field shows, there is proper whitespace to the tab bar below. But the default state has insufficient whitespace.

**Comment by jancborchardt** [comment]
LGTM, now with proper padding!

@karlitschek Requesting backport since the tags field really really distracts from the Sharing input field, as confirmed by @schiessle, @oparoz and multiple others.

**Comment by karlitschek** [comment]
hmm. bigger change. but backport makes sense 👍

**Pull request merged**